SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/Connection/SpatialEventTracer.h"
+
+#include <inttypes.h>
+
+#include "Interop/Connection/SpatialTraceEventBuilder.h"
+#include "SpatialGDKSettings.h"
+#include <WorkerSDK/improbable/c_io.h>
+#include <WorkerSDK/improbable/c_trace.h>
+
+DEFINE_LOG_CATEGORY(LogSpatialEventTracer);
+
+using namespace SpatialGDK;
+using namespace worker::c;
```

Comment:
I don't think we need to use this anyway. If we are just going to blanket using the namespace this doesn't offer any extra safety and we may as well just pretend they aren't there.

Reply:
The extern "C" means that they are all addressable via the global namespace, even though they are really defined as worker::c::whatever.
As we don't do this for the rest of the worker sdk stuff I'm not too concerned with using worker::c everywhere. However if we want to switch the style to that I'd be happy to do so. But IDEs probably won't help you with it.

Reply:
You are saying we should remove SpatialEventTracer from the SpatialGDK namespace? What should the SpatialGDK namespace contain?

Reply:
No everything should be in the SpatialGDK namespace. We don't need to care about worker::c